model/common/src/icon4py/model/common/grid/geometry.py

```diff
@@ -438,6 +463,18 @@ def _register_computed_fields(self):
         )
         self.register_provider(tangent_cell_wrapper)
 
+        primal_cart_normal = factory.NumpyFieldsProvider(
```

Comment:
I thinks this one you don't need it, it just combines the x,y,z components, but in the end we cannot use those cartesian vectors because gt4py only supports scalar fields.